Formula/s/siril.rb

```diff
@@ -60,9 +58,9 @@ def install
     ENV.append_to_cflags "-I#{HOMEBREW_PREFIX}/include"
     ENV.append_to_cflags "-Xpreprocessor -fopenmp -lomp" if OS.mac?
 
-    system "./autogen.sh", "--prefix=#{prefix}"
-    system "make"
-    system "make", "install"
+    system "meson", "setup", *std_meson_args, "_build"
+    system "ninja", "-C", "_build"
+    system "ninja", "-C", "_build", "install"
```

Comment:
we can just do this these days

```
    system "meson", "setup", "build", *std_meson_args
    system "meson", "compile", "-C", "build", "--verbose"
    system "meson", "install", "-C", "build"
```

Reply:
I saw that in other formulae as well as [upstream](https://gitlab.com/free-astro/siril_macos/-/blob/master/210-siril_build.sh) but wasn't sure what the difference was (i.e., both the `ninja` and `meson` approaches seemed to work the same). I'll update this in a moment.

Reply:
This would be more of code style :)